Gemfile.next.lock

```diff
@@ -185,7 +195,12 @@ GEM
     crass (1.0.6)
     css_parser (1.7.1)
       addressable
-    database_cleaner (1.7.0)
+    database_cleaner (2.0.2)
```

Comment:
We updated `database_cleaner` because we were getting this error in Rails 7.1: 

```
An error occurred in a `before(:suite)` hook.
Failure/Error: DatabaseCleaner.clean_with(:deletion)

NoMethodError:
  undefined method `table_name' for ActiveRecord::SchemaMigration:Class
# ./spec/rails_helper.rb:161:in `block (2 levels) in <top (required)>'
```